modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testIntegration/java/com/liferay/headless/admin/taxonomy/resource/v1_0/test/KeywordResourceTest.java

```diff
@@ -269,4 +309,101 @@ protected Keyword testGetKeywordsRankedPage_addKeyword(Keyword keyword)
 		return keyword;
 	}
 
+	protected Keyword testGraphQLAssetLibraryKeyword_addKeyword(Keyword keyword)
+		throws Exception {
+
+		JSONDeserializer<Keyword> jsonDeserializer =
```

Comment:
The purpose of this method is to provide a way to create a keyword for the testGraphQLAssetLibrary test. So you don't need to add this implementation. I mean, this method does not need to use GraphQL at all. So I would say, maybe better to use the same pattern as in the `testGetAssetLibraryKeywordByExternalReferenceCode_addKeyword` method

Reply:
I do not understand this, I will ask you more information about it.